LICENSES/EE-used-libs.json

```diff
@@ -0,0 +1,626 @@
+[
+    {
+        "library": "alembic",
+        "version": "1.12.0"
+    },
+    {
+        "library": "amqp",
+        "version": "5.1.1"
+    },
+    {
+        "library": "async-timeout",
+        "version": "4.0.3"
+    },
+    {
+        "library": "attrs",
+        "version": "23.1.0"
+    },
+    {
+        "library": "bcrypt",
+        "version": "4.0.0"
+    },
+    {
+        "library": "billiard",
+        "version": "3.6.4.0"
+    },
+    {
+        "library": "binaryornot",
+        "version": "0.4.4"
+    },
+    {
+        "library": "blinker",
+        "version": "1.6.2"
+    },
+    {
+        "library": "cachelib",
+        "version": "0.9.0"
+    },
+    {
+        "library": "celery",
+        "version": "5.2.7"
+    },
+    {
+        "library": "certifi",
+        "version": "2023.7.22"
+    },
+    {
+        "library": "chardet",
+        "version": "5.2.0"
+    },
+    {
+        "library": "charset-normalizer",
+        "version": "3.3.0"
+    },
+    {
+        "library": "click",
+        "version": "8.1.7"
+    },
+    {
+        "library": "click-didyoumean",
+        "version": "0.3.0"
+    },
+    {
+        "library": "click-plugins",
+        "version": "1.1.1"
+    },
+    {
+        "library": "click-repl",
+        "version": "0.3.0"
+    },
+    {
+        "library": "clickclick",
+        "version": "20.10.2"
+    },
+    {
+        "library": "colorama",
+        "version": "0.4.5"
+    },
+    {
+        "library": "connexion",
+        "version": "2.14.1"
+    },
+    {
+        "library": "Deprecated",
```

Comment:
this is library?

Reply:
it is also included in pipfile.lock next to other libs